Source/WebCore/page/Quirks.cpp

```diff
@@ -1091,7 +1091,26 @@ bool Quirks::isMicrosoftTeamsRedirectURL(const URL& url)
     return url.host() == "teams.microsoft.com"_s && url.query().toString().contains("Retried+3+times+without+success"_s);
 }
 
-// microsoft.com rdar://72453487
```

Comment:
Could you move the comment please? back underneath :)

Reply:
The radar is for "Can't login to Skype from Microsoft Outlook account in Safari" (https://trac.webkit.org/changeset/271074/webkit), which is fixed by:
```
// Skype case.
--
// FIXME(220105): Remove this quirk once Skype under outlook.live.com completes their login flow redesign.
if (url.host() == "outlook.live.com"_s) {
return element.hasClass()
&& (element.classNames().contains("_3ioEp2RGR5vb0gqRDsaFPa")
\|\| element.classNames().contains("_2Am2jvTaBz17UJ8XnfxFOy"));
}
```
and https://bugs.webkit.org/show_bug.cgi?id=220105 is already fixed, so I just dropped it.